WebUI/static/js/test.js

```diff
@@ -0,0 +1,12 @@
+document.onload = last100;
```

Comment:
Is this file part of a unit test? This needs to be clarified in the file with comments / file name.